clmm/theory/func_layer.py

```diff
@@ -560,8 +562,8 @@ def compute_convergence(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo, delt
 def compute_reduced_tangential_shear(
         r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
         delta_mdef=200, halo_profile_model='nfw', massdef='mean', alpha_ein=None,
-        z_src_model='single_plane', beta_s_mean=None, beta_s_square_mean=None,
-        z_distrib_func=None, validate_input=True, verbose=False):
+        z_src_model='discrete', beta_s_mean=None, beta_s_square_mean=None,
+        z_distrib_func=None, gt_equation='applegate14', validate_input=True, verbose=False):
```

Comment:
I know the tests are failing, but changing `gt_equation='applegate14'` to ` gt_equation=None` fixes a failed test from `test_mockdata.py`.

Reply:
Ok. Not very sure why we default `gt_equation` to `'applegate14'`, which is not compatible with `z_src_model='discrete'`. The failed test should probably be fixed from `test_mockdata.py` itself.